mock-connector/src/main/scala/it/agilelab/darwin/connector/mock/MockConnector.scala

```diff
@@ -70,28 +70,30 @@ class MockConnector(config: Config) extends Connector with Logging {
     }
   }
 
-  private val table: mutable.Map[Long, Schema] = {
-    val m = mutable.Map.empty[Long, Schema]
+  private val table: mutable.Map[Long, Schema] = mutable.Map.empty[Long, Schema]
+
+  override def fullLoad(): Seq[(Long, Schema)] = {
```

Comment:
Since fullLoad() is called each time a find is issued, it could check if map is empty and load resources only in that case (since configuration cannot change, once data is loaded there is no need to load it again).

Reply:
Shouldn't instead avoid to call it at every find? 🤔

Reply:
No, that was dumb. Anyway that is a matter of the manager/cache, the connector in that case it is "expected" to perform some IO to get the result. Isn't it? I wouldn't worry too much about performance here.